gen-compdocs/generators/doc.go

```diff
@@ -107,13 +107,13 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 		// Print the "generated" warning
 		fmt.Fprintf(w, "%s\n\n", generated_warning)
 
-		if _, err := fmt.Fprintf(w, "%s\n\n", "## {{% heading \"synopsis\" %}}"); err != nil {
+		if _, err := fmt.Fprintf(w, "%s\n\n", `## {{% heading "synopsis" %}}`); err != nil {
```

Comment:
I changed the quoting here because using plain `"..."` broke the syntax highlighting in VSCode.